<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userlogin</class>
 <widget class="QDialog" name="userlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/new/prefix1/src/icons8-c-50.png</normaloff>
    <normalon>:/new/prefix1/src/icons8-c-50.png</normalon>:/new/prefix1/src/icons8-c-50.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>531</width>
     <height>336</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>36</number>
      </property>
      <item>
       <widget class="QLabel" name="loginlabel">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 600 12pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:700;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="margin">
         <number>18</number>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="username_label">
          <property name="styleSheet">
           <string notr="true">font-color : rgb(255,255, 255);
font: 700 14pt &quot;Script MT Bold&quot;;
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; font-style:italic;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="username_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(34, 38, 41);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="password_label">
          <property name="styleSheet">
           <string notr="true">font-color : rgb(255,255, 255);
font: 700 14pt &quot;Script MT Bold&quot;;
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; font-style:italic;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password_lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(34, 38, 41);</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>32</number>
      </property>
      <item>
       <widget class="QPushButton" name="login_button">
        <property name="styleSheet">
         <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgb(0, 0, 0);
background-color: rgb(211, 217, 212);
</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signup_button">
        <property name="styleSheet">
         <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgb(0, 0, 0);
background-color: rgb(211, 217, 212);
</string>
        </property>
        <property name="text">
         <string>Signup</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
